src/main/java/org/commcare/suite/model/AppAvailableForInstall.java

```diff
@@ -0,0 +1,73 @@
+package org.commcare.suite.model;
+
+import org.javarosa.core.util.externalizable.DeserializationException;
+import org.javarosa.core.util.externalizable.ExtUtil;
+import org.javarosa.core.util.externalizable.Externalizable;
+import org.javarosa.core.util.externalizable.PrototypeFactory;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.Serializable;
+
+/**
+ * Represents an app that exists on CommCare HQ (either india or prod) and is accessible for
+ * the currently-authenticated user (either mobile or web) to install.
+ *
+ * Created by amstone326 on 2/3/17.
+ */
+public class AppAvailableForInstall implements Serializable, Externalizable {
+
+    private String domain;
+    private String appName;
+    private String appVersion;
+    private boolean isOnProd;
+    private String profileRef;
+    private String mediaProfileRef;
+
+    public AppAvailableForInstall() {
+        // for deserialization
+    }
+
+    public AppAvailableForInstall(String domain, String appName, String appVersion,
+                                  boolean isOnProd, String profileRef, String mediaProfileRef) {
+        this.domain = domain;
+        this.appName = appName;
+        this.appVersion = appVersion;
+        this.isOnProd = isOnProd;
```

Comment:
Yea.. wasn't sure what to do about this; I have all of these here because Farid is sending down all of this info in the response, so I felt like I should store it in case it became useful later. But I guess YAGNI, and I should just ignore the fields that I'm not using.